<ui version="4.0" >
 <class>EditAutoClassifyLibraryClass</class>
 <widget class="QDialog" name="EditAutoClassifyLibraryClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EditAutoClassifyLibrary</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetLibList" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>221</width>
     <height>311</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text" >
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frameLibFolder" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>名称</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Library Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibDelete" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibSaveNew" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Save New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibSave" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibID" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>12345</string>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibNew" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Clear new</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>200</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_14" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>輪郭からの間隔</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditSpaceToOutline" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ノイズサイズ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditNoiseSize" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_16" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>優先度</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditPriority" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_33" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text" >
     <string>生
成</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent" >
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>50</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QSpinBox" name="EditMinBlockSize" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ブロック最大サイズ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMaxBlockSize" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ブロック最小サイズ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinBlockDots" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>2000000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ブロック最小面積</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMaxBlockDots" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>2000000000</number>
    </property>
    <property name="value" >
     <number>2000000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ブロック最大面積</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>限界サイズ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditLimitBlockSize" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_32" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text" >
     <string>サ
イ
ズ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent" >
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
